### PR TITLE
A few things here and there II

### PR DESCRIPTION
**Pull request by sgeto**

### CODE REVIEW

**Comment by sgeto** [comment]
I usually place the DLL in a different directory in my path to not depend on a VS developer command prompt (by altering `BIN_TARGET`). To be honest I place them it system32.

The next best things would be to have a static version of wsock-trace. Any plans on that?

**Comment by sgeto** [comment]
You're right about nmake. It sucks.

File: src/Makefile.vc6 [hunk 1]
```diff
@@ -126,7 +126,7 @@ $(WSOCK_DLL) $(WSOCK_LIB): $(WSOCK_DEF) $(OBJECTS) $(NON_EXPORT_OBJ) compile_jua
 	@type link.tmp >> $(WSOCK_MAP)
 	@del $(WSOCK_LIB:.lib=.exp)
 
-install:
+install: all
```
Comment (gvanem) [inline src/Makefile.vc6]: That's okay. But now that `install` is not part of `all`, I'll removed the check for `"$(USER)" != "AppVeyor"`.

**Comment by gvanem** [comment]
I'm still a git noob. So I'll merge your changes here locally and do a:
```
git commit --author autostart.ini@gmail.com Makefile.vc6
```

How am I supposed to do this w/o my local repo becoming out-of-sync with *master*?

**Comment by sgeto** [comment]
> How am I supposed to do this w/o my local repo becoming out-of-sync with master?

I never had to do that but the maintainer of netmap merged some of my commit by importing them from my "remote tracking branch" (aka fork) rather than the PR I send. I think that's a safest approach. I never tried this though.

**Comment by sgeto** [comment]
Honestly your way is fine with me too. Whatever works really...

File: src/Makefile.vc6 [hunk 2]
```diff
@@ -182,6 +182,23 @@ vclean realclean: clean
 	-del idna.exe  idna.map  idna.pdb  test.exe     test.pdb
 	-rd $(OBJ_DIR)
 
+help:
+	@echo Run one of the following commands:
+	@echo
+	@echo nmake -f Makefile.vc6 VAR=VAL
+	@echo nmake -f Makefile.vc6 install VAR=VAL
+	@echo
```
Comment (gvanem) [inline src/Makefile.vc6]: I'd rather make these indented:
```
	@echo "  nmake -f Makefile.vc6 VAR=VAL"
	@echo "  nmake -f Makefile.vc6 install VAR=VAL"
```
I tested with `cmd.exe`, `powershell.exe` and `4nt.exe` as `COMSPEC`. And the extra `"` don't show up.

Comment (sgeto) [reply src/Makefile.vc6]: 😆 I tried too. I have no idea how to get indentation this way...
Maybe escape the spaces? `\ \ \ \ \`

Comment (gvanem) [reply src/Makefile.vc6]: What do you mean? How is this printed in your shell:
```
help:
	@echo Run one of the following commands:
	@echo "  nmake -f Makefile.vc6 VAR=VAL"
	@echo "  nmake -f Makefile.vc6 install VAR=VAL"
	@echo
	@echo where VAR can be of the following:
	@echo "  USE_DEBUG       (default: $(USE_DEBUG))"
	@echo "  USE_DEF_FILE    (default: $(USE_DEF_FILE))"
	@echo "  USE_IP2LOCATION (default: $(USE_IP2LOCATION))"
	@echo "  USE_LUA         (default: $(USE_LUA))"
	@echo "  LUAJIT_ROOT     (default: '$(LUAJIT_ROOT)')"
	@echo "  WANT_64_BITS    (default: check)"
	@echo "  LIB_TARGET      (default: '$(LIB_TARGET)')"
	@echo "  BIN_TARGET      (default: '$(BIN_TARGET)')"
```
I get perfect result:
```
Run one of the following commands:
  nmake -f Makefile.vc6 VAR=VAL
  nmake -f Makefile.vc6 install VAR=VAL

where VAR can be of the following:
  USE_DEBUG       (default: 0)
  USE_DEF_FILE    (default: 1)
  USE_IP2LOCATION (default: 1)
  USE_LUA         (default: 0)
  LUAJIT_ROOT     (default: 'f:\MingW32\src\Parsers\LuaJIT')
  WANT_64_BITS    (default: check)
  LIB_TARGET      (default: 'f:\gv\VC_2017\lib')
  BIN_TARGET      (default: 'f:\gv\VC_2017\bin')
```

Comment (sgeto) [reply src/Makefile.vc6]: oh okay. This works. I didn't work for me. So I gave up.

Comment (gvanem) [reply src/Makefile.vc6]: Okay. What about this:
```
help:
	@type <<
Run one of the following commands:
  nmake -f Makefile.vc6 VAR=VAL
  nmake -f Makefile.vc6 install VAR=VAL

where VAR can be of the following:
  USE_DEBUG       (default: $(USE_DEBUG))
  USE_DEF_FILE    (default: $(USE_DEF_FILE))
  USE_IP2LOCATION (default: $(USE_IP2LOCATION))
  USE_LUA         (default: $(USE_LUA))
  LUAJIT_ROOT     (default: '$(LUAJIT_ROOT)')
  WANT_64_BITS    (default: check)
  LIB_TARGET      (default: '$(LIB_TARGET)')
  BIN_TARGET      (default: '$(BIN_TARGET)')
<<
```

Comment (sgeto) [reply src/Makefile.vc6]: yes, this is even better.

**Comment by gvanem** [comment]
Closed via https://github.com/gvanem/wsock-trace/commit/8d7870f54afe56d99e3df58a2e9d596737d18e71

Thanks!

**Comment by gvanem** [comment]
> The next best things would be to have a static version of wsock-trace. Any plans on that?

That would be more difficult. Without any `DLL_PROCESS_ATTACH`.
Would need to do all initialisation from `WSAStartup()`.

**Comment by sgeto** [comment]
Oh... I see

**Pull request closed without merging**